content/en/references/configuration.md

```diff
@@ -406,16 +406,16 @@ These configurations have already been removed and **won't have any effect** on
 | `LEGACY_DIRECTORIES` | 2.0.0 | `0` (default) | Use legacy method of managing internal filesystem layout. See [Filesystem Layout]({{< ref "filesystem" >}}). |
 | `LEGACY_INIT_DIR` | 2.0.0 | `1` \| `0`(default) | Used with `INIT_SCRIPTS_PATH`. This has been replaced by the [init-hook system](https://docs.localstack.cloud/references/init-hooks/). |
 | `MULTI_ACCOUNTS` | 2.0.0 | `0` (default) | Enable multi-accounts (preview) |
-| `REQUIRE_PRO` | 2.0.0 | 0 (default) | Whether to require license activation to succeed to start LocalStack. If set to 0 (default) LocalStack will start as community version if the license cannot be activated. |
+| `REQUIRE_PRO` | 2.0.0 | 0 (default) | Whether to require license activation to succeed to start LocalStack. If set to 0 (default) LocalStack will start as Community image if the license cannot be activated. |
```

Comment:
Need some input from core team /cc: @alexrashed

Reply:
That is correct, there is no `REQURE_PRO` (there might have been sometime in the past or this is a typo?), there only is `ACTIVATE_PRO` which defaults to `1`.

Reply:
Chiming in here: Before the community / pro image split there was a `REQUIRE_PRO`, basically if the activation failed for some reason, it would start community. With `REQUIRE_PRO=1` , it would instead shut LocalStack down, to immediatly show the problem.
Since the split, this is the default in the pro image anyway, so the variable is not used anymore.